AppOne/AppOne/Model/Brands.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Brands.swift
+//  AppOne
+//
+//  Created by Asapu, Ajaikumar on 20.11.22.
+//
+
+struct Brands : Hashable {
+    let name: String
+    let headQuarters: String
+    let ceo: String
+
+    func hash(into hasher: inout Hasher) {
+        hasher.combine(name)
+    }
```

Comment:
can be removed, compiler synthesizes it, also the hash only over the name is not very reliable